src/main/java/winstone/cmdline/Option.java

```diff
@@ -80,6 +80,10 @@
     public static final OInt CONTROL_PORT=integer("controlPort",-1);
 
     public static final OBoolean USE_JMX=bool("useJmx",false);
+    public static final OBoolean USE_QTP=bool("useQTP",false);
```

Comment:
I would definitely prefer having true per default here:
``` public static final OBoolean USE_QTP=bool("useQTP",true); ```

Reply:
I am fine with that. CC @jenkinsci/code-reviewers, more votes would be helpful